packages/devtools_app/pubspec.yaml

```diff
@@ -30,11 +30,11 @@ dependencies:
   devtools_extensions: ^0.0.10
   devtools_shared: ^6.0.1
   file: ">=6.0.0 <8.0.0"
-  file_selector: ^0.8.0
-  file_selector_linux: ^0.0.2
-  file_selector_macos: ^0.0.2
-  file_selector_web: ^0.8.1
-  file_selector_windows: ^0.0.2
+  file_selector: ^1.0.0
+  file_selector_linux: ^0.9.0
+  file_selector_macos: ^0.9.0
+  file_selector_web: ^0.9.0
+  file_selector_windows: ^0.9.0
```

Comment:
Is devtools interacting directly with any implementation package? If not, none of these except `file_selector` is still necessary; they are all endorsed by file_selector 1.0.0.

Reply:
Sure! I was just riffing on what's there. Zero clue about why all of these show up!

Reply:
The original dependencies date back to before `file_selector`'s implementation had moved and been endorsed, IIRC.

Reply:
We only import `file_selector` in the DevTools code. Can we remove all these platform specific imports then?

Reply:
Yes, [all the implementations are already transitive dependencies now](https://github.com/flutter/packages/blob/ca16173e67e42076996f1f9957479d8ad847c80b/packages/file_selector/file_selector/pubspec.yaml#L29-L35).

Reply:
Addressed in https://github.com/flutter/devtools/pull/6895